bin/addJars.hql

```diff
@@ -0,0 +1 @@
+ADD JAR /usr/share/fk-bigfoot-hivejsonserde/json-serde-1.3-SNAPSHOT-jar-with-dependencies.jar; ADD JAR /usr/share/fk-bigfoot-dimlookup/dimlookup-hive-udf-1.0-SNAPSHOT-jar-with-dependencies.jar;
```

Comment:
where did these extra jar come from ? hive server will already have them right?

Reply:
Yeah but the jars should be added when beeline execute, currently hive cli by defaults add these configs.